sdk/core/test/integration/otp/index.test.ts

```diff
@@ -311,6 +313,27 @@ describe('CommonNetworkMember (flows that require OTP)', () => {
     expect(responseError.name).to.eql('COR-17')
   })
 
+  it('#passwordlessLogin with custom messages ', async () => {
+    const stamp = `${Date.now()} stamp`
+    const delimiter = ':'
+    const messageParameters: MessageParameters = {
+      message: `Your verification code is ${delimiter}${stamp}${delimiter}{{CODE}}.`,
+      subject: `${stamp}${delimiter}{{CODE}}`,
+    }
+
+    const fullOptions = getOptionsForEnvironment(true)
+    const tag = `${fullOptions.env}_passwordlessLogin_integration`
+    const userName = TestEmailHelper.generateEmailForTag(tag)
+    const token = await CommonNetworkMember.passwordlessLogin(userName, fullOptions, messageParameters)
+    await wait(DELAY)
+    const [{ text, subject }] = await TestEmailHelper.getEmailsForTag(tag)
+    const [_, expectedStampFromMessage, __] = text.split(delimiter)
+    const [expectedStampFromSubject, otp] = subject.split(delimiter)
+    expect(expectedStampFromMessage).to.equal(stamp)
+    expect(expectedStampFromSubject).to.equal(stamp)
+    await CommonNetworkMember.completeLoginChallenge(token, otp, fullOptions)
```

Comment:
why do we need to complete login challenge as there are no expects afterwards?

Reply:
to validate OTP code extracted from template

Reply:
Maybe then add `expect(response.accessToken).to.exist` to make the intent clear?